rpcserver.go

```diff
@@ -5659,13 +5660,25 @@ func handleVerifyMessage(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 
 // handleVersion implements the version command.
 func handleVersion(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	buildMeta := normalizeBuildString(runtime.Version())
```

Comment:
would this cause a runtime version e.g. `go1.10` to enter the build metadata, when build metadata should be separated by periods?  This would result in two additional metadata identifiers, `go1` and `10`, which is not what we intend here.

Reply:
Yeah, that's what I was going for.  I suppose I could replace the period with a an -.

Reply:
I'm open to other suggestions here too.  Basically, I want to be able to see the runtime information as part of the build metadata.  Showing it as `go110` doesn't work.  Given the alphabet only allows `[0-9a-zA-Z-]` the only other option is `go1-10`.  Currently it shows up as:

```JSON
{
  "dcrd": {
    "versionstring": "1.1.2+blockchainremovemainchainindex",
    "major": 1,
    "minor": 1,
    "patch": 2,
    "prerelease": "",
    "buildmetadata": "blockchainremovemainchainindex.go1.10"
  },
  "dcrdjsonrpcapi": {
    "versionstring": "3.2.0",
    "major": 3,
    "minor": 2,
    "patch": 0,
    "prerelease": "",
    "buildmetadata": ""
  }
}

```

Reply:
Update to replace `.` with `-` then pass it through the semver build normalizer.